rust/private/rustc.bzl

```diff
@@ -76,6 +77,14 @@ def _get_rustc_env(ctx, toolchain):
         "CARGO_CFG_TARGET_ARCH": toolchain.target_arch,
     }
 
+    # Add any additional explicitly passed environment variables.
+    extra_envs = []
+    for kv in getattr(ctx.attr, "rustc_env", []):
+      k, v = kv.split("=")
```

Comment:
This will fail for values that contain an `=` character, such as `A=B=C`, which chould be parsed to `["A", "B=C"]`. This could be fixed with
```python
k, v = kv.split("=", 1)
```